apps/server/src/modules/auth/next-auth.controller.ts

```diff
@@ -140,6 +147,29 @@ export class NextAuthController {
       }
     }
 
+    let nextAuthTokenCookie: (CookieOption & { value: string }) | undefined;
+    // next-auth credentials login only support JWT strategy
+    // https://next-auth.js.org/configuration/providers/credentials
+    // let's store the session token in the database
+    if (
+      credentialsSignIn &&
+      (nextAuthTokenCookie = cookies?.find(
+        ({ name }) => name === 'next-auth.session-token'
```

Comment:
Need to add Secure_ prefix when https is enabled

Reply:
Fix it for me